test/TesterInternal/MembershipTests/DynamoDBMembershipTableTest.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Orleans;
+using Orleans.Messaging;
+using Orleans.Runtime;
+using Orleans.Runtime.MembershipService;
+using System.Threading.Tasks;
+using UnitTests.StorageTests.AWSUtils;
+using Xunit;
+
+namespace UnitTests.MembershipTests
+{
+    /// <summary>
+    /// Tests for operation of Orleans Membership Table using AWS DynamoDB - Requires access to external DynamoDB storage
+    /// </summary>
+    public class DynamoDBMembershipTableTest : MembershipTableTestsBase, IClassFixture<DynamoDBStorageTestsFixture>
```

Comment:
It should fail quickly. However, it is not so easy as in Azure or SQL to detect if DynamoDB local instance is "installed" since there is nothing to install. So basically all the tests will quickly fail as they can't connect to DynamoDB. Hard to tell  it to skip and that is why it is (as any other provider) not a BVT test and has its own category so people going to test it make sure to have the correct infrastructure before start.